.github/workflows/quickstart.yml

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+# Workflow to automatedly verify the quickstart instructions
+
+name: Verify Quickstart
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 15 * * *"
+
+jobs:
+  iso_quickstart:
+    runs-on: ubuntu-18.04
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.2
+      with:
+        ref: '1.0-stable'
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: Install Remaining Prerequisites
+      run: |
+        # Golang and docker are already installed on the agent
+        sudo apt-get update
+        sudo apt -y install make tar wget curl rpm qemu-utils genisoimage pigz
+
+    - name: Configure the Environment
+      run: |
+        pushd toolkit
+        sudo make go-tools REBUILD_TOOLS=y
+        sudo make input-srpms DOWNLOAD_SRPMS=y
```

Comment:
I'm torn on splitting these off, they are definitely one-time operations and I think they fit fine in the setup area of the readme, but splitting them might mask an issue with the makefile...

Reply:
I intended these tasks to run exactly what is written in `building.md`, modulo some environment "prerequisite" setup. Your concerns are valid though. We could set up another Action to test for issues with the makefile, especially if the makefile changes.